README.md

```diff
@@ -188,8 +188,8 @@ Here the returned value is a number as `digit` and `natural` have the
 ### `seq`
 
 For a simple sequencing of parsers, use the
-`seq(parsers: Parser<?>[]): Parser<?[]>` combinator. The input parsers can have
-different types, which will be reflected in the resulting parser
+`seq(...parsers: Parser<?>[]): Parser<?[]>` combinator. The input parsers can
```

Comment:
This one is not a typo since `seq` is variadic

Reply:
This is just the function declaration in code. Without the spread operator it’s a single array argument.

Reply:
Oops my bad I mingled the + and - while skimming over this